.circleci/config.yml

```diff
@@ -46,6 +46,9 @@ commands:
             . venv/bin/activate
             pip install --upgrade pip wheel
             pip install -r ./test_requirements/requirements_<<parameters.py>>_optional.txt
+            cd js
+            npm install
```

Comment:
Please consider using `npm ci` instead of `npm i` everywhere to have more stable packages as well as being faster on CircleCI.